src/mainframe/endpoints/report.py

```diff
@@ -121,20 +120,17 @@ def _validate_additional_information(body: ReportPackageBody, scan: Scan):
     log = logger.bind(package={"name": body.name, "version": body.version})
 
     if body.additional_information is None:
-        if len(scan.rules) == 0 or body.use_email is False:
-            if len(scan.rules) == 0:
-                detail = (
-                    f"additional_information is a required field as package "
-                    f"`{body.name}@{body.version}` has no matched rules in the database"
-                )
-            else:
-                detail = "additional_information is required when using Observation API"
-
-            error = HTTPException(400, detail=detail)
-            log.error(
-                "Missing additional_information field", error_message=detail, tag="missing_additional_information"
+        if len(scan.rules) == 0:
```

Comment:
This is a nitpick, but generally you should do `if not sequence:` for sequences instead of checking the length.